test/H5Test.cpp

```diff
@@ -930,87 +879,64 @@ TEST_CASE("H5 Utilities")
   REQUIRE(objectName == "Data");
 }
 
+#if 0
```

Comment:
Is the issue we had with implicit copies no longer an issue?

Reply:
This should be uncommented since it was a test on the current 'develop' branch